### DISCUSSION TITLE
Dynamic Placement of Nodes

### DISCUSSION THREAD

**Discussion by amitjoy** [eckig/graph-editor#104] [Q&A]
Is there any helper to dynamically place nodes on the graph? Sometimes, it is useful to create a graph dynamically and it would be easier to place them automatically using node placement strategies.

**Comment by eckig** [comment]
I tested integrating the Eclipse Layout Kernel (ELK) and it worked fine. But I never found the time to do it right..

For reference: 

- The ELK project: https://github.com/eclipse/elk
- There is a feature branch `elk` with some initial work.. more like a playground.
